telemetry/specs/telemetry_report.mdk

```diff
@@ -439,54 +456,78 @@ The Destination Port is user configurable. The expectation is that the same
+  : Identifies the hardware subsystem within the node that generated
     this report. For example, in a chassis with multiple linecards this could
     identify a specific linecard, or a subsystem within a linecard. The hw_id
-    is unique within the scope of a Node id.
+    is unique within the scope of a Node ID.
 
 * Sequence Number (22b): Sequence Number
   : Reflects the sequence of reports from a specific combination of
-    (Node id, hw_id) to a particular telemetry report destination. This
+    (Node ID, hw_id) to a particular telemetry report destination. This
     can be used to detect loss of telemetry reports before they reach their
     intended destination.
 
 * Node ID (32b): Node ID
-  : The unique ID of a network device (such as routers, switches, NICs etc.) This is generally administratively assigned. Node IDs
-    must be unique within a management domain.
+  : The unique ID of a node. This is generally administratively assigned.
+    Node IDs must be unique within a management domain.
+
+## Individual Report Header (Ver 2.0) (4+ octets) { #individual-report }
+
+Each telemetry report packet contains one or more reports immediately
+following the Telemetry Reports Outer Header. Each report within the packet
+starts with the Individual Report Header. The presence of multiple reports
+corresponding to multiple data plane packets, possibly from multiple flows,
+can be determined by comparing the *Report Length* in the Individual Report
+Header with the length in the UDP header.
+
+```
+   0                   1                   2                   3
+   0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+  |RepType| InType| Report Length |   MD Length   |D|Q|F|I|  Rsvd |
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+<--+
+  |                                                               |   |
+  |    Telemetry Report Contents (varies depending on RepType)    |   |
+  |                                                               |   |
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ Report
+  |                                                               | Length
+  |                     Inner Report Contents                     |   |
+  |       (Truncated Packet or Additional DS Extension Data       |   |
+  |                  or TLV depending on InType)                  |   |
+  |                                                               |   |
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+<--+
+```
 
 * RepType (4b): Report Type
   : Type of Telemetry report
-    * 0: INT v2.0 or later.
-    * 1: IOAM
-    * 2 – 15: Reserved
+    * **0**: Inner Only
+    * **1**: INT
+    * **2**: IOAM
+    * **3 – 15**: Reserved
 
 * InType (4b): Inner Type
-  : Type of data embedded after Variable Optional Baseline & DS Metadata.
+  : Type of data embedded after Telemetry Report Contents:
```

Comment:
how about "... Individual Report Contents"

Reply:
I see there are multiple occurrences of this term below. Would need to replace all.